pkg/grafana/grafana.go

```diff
@@ -102,6 +109,46 @@ func DeleteByID(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI, id int64
 	return nil
 }
 
+func CreateDatasource(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI) ([]Datasource, error) {
+	logger := log.FromContext(ctx)
+
+	createdDatasources := make([]Datasource, 0)
+	mimirDatasource := Datasource{
+		Name:   MimirDatasourceName,
+		Type:   MimirDatasourceType,
+		Url:    MimirDatasourceUrl,
+		Access: DatasourceAccessMode,
+	}
+	lokiDatasource := Datasource{
+		Name:   lokiDatasourceName,
+		Type:   LokiDatasourceType,
+		Url:    LokiDatasourceUrl,
+		Access: DatasourceAccessMode,
+	}
+
+	logger.Info("creating datasources")
+	for _, toCreateDatasource := range []Datasource{mimirDatasource, lokiDatasource} {
+		createdDataSource, err := grafanaAPI.Datasources.AddDataSource(&models.AddDataSourceCommand{
```

Comment:
I would make the whole models.AddDatasourceCommand a function because we will use it for updates right?

Reply:
But then it won't be of `AddDatasourceCommand` type but of `UpdateDatsourceCommand` so we can't create a straight forward function.